File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        for(String s : babbling){
            s = s.replace("aya", ".").replace("ye", ".").replace("woo", ".").replace("ma", ".");
            if(s.replace(".","").equals(""))
                answer++;
        }
        return answer;
    }
}

File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    var answer = 0;
    for(var str of babbling){
        str = str.replace(/aya|ye|woo|ma/g, " ");
        str = str.replace(/ /g, '');
        if(str === ''){
            answer++;
        }
    }
    return answer;
}